File: submit.py
#!/usr/bin/python3.6

import cgi
import smtplib

import cgitb
cgitb.enable()

parameters = cgi.FieldStorage()
try:
    first_name = parameters['first_name'].value
    last_name = parameters['last_name'].value
    email = parameters['email'].value
    comment = parameters['comment'].value
    print("Content-type:text/html\r\n\r\n")
    print(first_name, last_name, email, comment)

    body = """From: Weisz.io Contact Form <no-reply@weisz.io>
To: Michael Weisz <mail@michael-weisz.de>
MIME-Version: 1.0
Content-type: text/html
Subject: Message via weisz.io contact form

First Name: {} <br>
Last Name: {} <br>
Email: {} <br>
Message: {}""".format(first_name, last_name, email, comment)

    server = smtplib.SMTP('localhost')
    server.sendmail('sender@address8912309123.com', 'mail@michael-weisz.de', body)
    server.quit()
except KeyError:
    print('Status: 400 Bad Request')
    exit(1)
except SMTPException:
    print('Status: 500 Bad Request')
    exit(1)

File: html/submit.php
<?php 
    $first_name = $_POST['first_name'];
    $last_name = $_POST['last_name'];
    $email = $_POST['email'];
    $comment = $_POST['comment'];

    $from = 'Michael-Weisz.de Contact Form: '; 
    $to = 'mail@michael-weisz.de'; 
    $subject = "Message from <$first_name $last_name> via michael-weisz.de contact form.";

    $body = "First Name: $first_name\n Last Name: $last_name\n E-Mail: $email\n Message:\n $comment";

    mail ($to, $subject, $body, $from)
?>